amoro-ams/src/main/java/org/apache/amoro/server/table/DefaultTableService.java

```diff
@@ -344,18 +346,35 @@ public List<Blocker> getBlockers(TableIdentifier tableIdentifier) {
   }
 
   @Override
-  public List<TableRuntimeMeta> getTableRuntimes(
+  public Pair<List<TableRuntimeMeta>, Integer> getTableRuntimes(
```

Comment:
Can you add some comments to explain the meaning of this return value?

Reply:
I've added the comments in the Interface, please take another look when you're free. thanks.